lib/manager/terraform/extract.ts

```diff
@@ -140,14 +166,15 @@ export function extractPackageFile(content: string): PackageFile | null {
       }
     } else if (
       dep.managerData.terraformDependencyType ===
-      TerraformDependencyTypes.provider
+        TerraformDependencyTypes.provider ||
+      dep.managerData.terraformDependencyType ===
+        TerraformDependencyTypes.required_providers
     ) {
       dep.depType = 'terraform';
       dep.depName = dep.managerData.moduleName;
       dep.depNameShort = dep.managerData.moduleName;
-      dep.managerData.lineNumber = dep.managerData.versionLine;
       dep.datasource = datasourceTerraformProvider.id;
-      if (dep.managerData.lineNumber) {
+      if (dep.managerData.versionLine) {
```

Comment:
Do we really need this check here? Or can we simply and remove version line completely? 🤔

Reply:
The only reason to still keep this is the more granular `skipReason.noVersion` error.     
This is a regular case with Terraform providers as defining a version is optional.

Reply:
@viceice So you still want me to remove it? I'm not sure at the moment ^^

Reply:
I think we can remove this check and leave the check on next line.

Reply:
I'm also not seeing why it's needed